RELEASE_NOTES.md

```diff
@@ -2,20 +2,20 @@
 
 ## Summary
 
-<!-- Here goes a general summary of what this release is about -->
-
 ## Upgrading
 
-<!-- Here goes notes on how to upgrade from previous versions, including deprecations and what they should be replaced with --> 
-
 - Add fine-grained Result types for the PowerDistributingActor.
     Previously Result was one class with many fields. Now each result has its own class
     that derives from Result parent class.
 
 ## New Features
 
-<!-- Here goes the main new features and examples or instructions on how to use them -->
+- Add inverter type to the Component data.
+    Inverter component has type that tells what kind of
+    inverter it is (Battery, Solar etc).
```

Comment:
In the release notes is better not to do manual wrapping because in the GitHub release section, damn GitHub will add a line break for each `\n`, so then it looks weird. Not sure if the extra newline I'm suggesting is necessary (or the 2 spaces indent), if you checked it and is fine like it is you can ignore it.

```suggestion
- Add inverter type to the Component data.

  Inverter component has type that tells what kind of inverter it is (Battery, Solar etc).
```

Reply:
I just change it to one line description (as it was made in last release). :)